pkg/api/const.go

```diff
@@ -434,10 +435,23 @@ const (
 const (
 	//AzureADIdentitySystem Identity System
 	AzureADIdentitySystem = "azure_ad"
-	//ADFS Identity System
+	//ADFSIdentitySystem System
 	ADFSIdentitySystem = "adfs"
 )
 
+const (
+	//Connected environment
+	Connected = "connected"
+	//AzureChina environment
+	AzureChina = "azure_china"
+	//AzureGerman environment
+	AzureGerman = "azure_german"
+	//AzureUSGovernment environment
+	AzureUSGovernment = "azure_us_government"
+	//DisConnect environment
+	DisConnect = "disconnected"
```

Comment:
```suggestion
	Disconnected = "disconnected"
```

Reply:
what is `disconnected `? I don't see it being used anywhere?

Reply:
for none ( or disconnected) we do not need download any dependencies. but still need an clould spec for set up the parameters for ARM template, so set it to AzurePublicCloud